rfc/reproducible-builds.md

```diff
@@ -91,3 +93,35 @@ buildah build \
 
 The result is an image with one deterministic layer (pinned by the hash) and deterministic build artifacts.
 Hence, the entire build is reproducible.
+
+### `apko` / `ko`
+To include c libraries into a distroless minimal image, we have to rebuild the this base image.
+For that, we can use `apko`.
+It can be configured using a `*.yaml` file and is easy to use. An exemplary image definition could look like this:
+
+```yaml
+contents:
+  repositories:
+    - https://dl-cdn.alpinelinux.org/alpine/edge/main
+    - https://dl-cdn.alpinelinux.org/alpine/edge/community  # for cryptsetup
+  packages:
+    - alpine-base
+    - cryptsetup-dev  # for disk-mapper
+    - gcompat         # for glibc/musl comparability
+
+environment:
+  PATH: /usr/sbin:/sbin:/usr/bin:/bin
+```
+
+Then in our `.ko.yaml`, we can use the newly created image as a base image, also just for certain build ids:
+```yaml
+baseImageOverrides:
+  github.com/edgelesssys/constellation/v2/disk-mapper/cmd: edgelesssys/alpine-libcryptsetup:base
```

Comment:
What are the `apko` commands you have used to get to this point?

I have tried to run `apko`, but it has a dependency on `apk`, so the easiest way is to run this build inside a container as well?

Reply:
Good point. I built the image using chainguard's docker image, I will mention that in the rfc.